jenkins-plugin/pom.xml

```diff
@@ -139,36 +139,6 @@
         <artifactId>symbol-annotation</artifactId>
         <version>${jenkins-symbol.version}</version>
```

Comment:
Can probably be removed, along with `annotation-indexer`.

Reply:
I just looked the BOMs, not the parent POM ... I will do another pass

Reply:
> Also, is there a reason you are not testing on Windows?

Historical, bad smell ...
Will check also